modules/programs/git.nix

```
@@ -265,7 +265,10 @@ in {
         genIdentity = name: account:
           with account;
           nameValuePair "sendemail.${name}" ({
-            smtpEncryption = if smtp.tls.enable then "tls" else "";
+            smtpEncryption = if smtp.tls.enable then
+              (if smtp.tls.useStartTls then "tls" else "ssl")
```

To maintain compatibility with existing configurations you can do something like
```suggestion
              (if smtp.tls.useStartTls || versionOlder config.home.stateVersion "20.09" then "tls" else "ssl")
```
and also add an entry to the release notes in `rl-2009.adoc`.